apps/example-app/.prettierignore

```diff
@@ -0,0 +1,6 @@
+node_modules
+ios
+android
+.vscode
+ignite/ignite.json
+package.json
```

Comment:
I'd just remove this file.

Reply:
Removed https://github.com/infinitered/reactotron/pull/1411/commits/89ab58464296f32d1900f7aafeed1b9576284c9d